.github/workflows/release.yml

```diff
@@ -23,8 +23,12 @@ on:
           You must use the non-abbreviated SHA (e.g. b2318b0 won't work!).
         required: true
         type: string
-      semantic_version:
-        description: The semver tag that you want to release (e.g. 0.52.1)
```

Comment:
Can you update the description on `dry_run` so that it's less confusing in the GitHub UI? Maybe just add "Dry run." to the beginning of it. In fact, basically all of these should start with a field name since the description is used as the label for the form input.

Reply:
Great suggestion. Updated like so in 3c8e69f:

![Screenshot 2023-10-24 at 10 19 07 PM](https://github.com/awslabs/smithy-rs/assets/15333866/097502e6-3193-43f6-b39b-dd231c2d14d7)